aerogear/digger-jenkins
# AGDIGGER-42 - general clean-up in docker setup

# Motivation

Cleanup configuration to include:

- Local build for the s2i image for image modifications
- Change directory to split between slave and jenkins image.

Note: image already pushed to aerogear repo. 

# Verification steps

Try to execute local build steps
Publish image to dockerhub (to check if you have permission to do so)
Setup new PersistentVolume
Change DeploymentConfig to include new image

# Expected results

Android node should be available by default.
